test/ibm/runtime/test_runtime_integration.py

```diff
@@ -141,6 +141,16 @@ def test_list_programs(self):
                 found = True
         self.assertTrue(found, f"Program {self.program_id} not found!")
 
+    def test_list_programs_with_program_name(self):
+        """Test listing programs with program name."""
+        program_id = self._upload_program(name="sample-program")
+        programs = self.provider.runtime.programs(name="sample-program")
```

Comment:
```suggestion
        program_id = self._upload_program(name="qiskit-test-sample")
        programs = self.provider.runtime.programs(name="qiskit-test-sample")
```
We already have an IBM provided public `sample-program`. So we don't want to mess with that name :)